### ISSUE TITLE
Indentation in multiline field initializer is wrong when working with arrays

### ISSUE THREAD

**Issue by jwosty** [mono/monodevelop#5222]
Code should be formatted like this:

```csharp
public MyClass
{
    public static int[] Field =
        new int[]
        {
            42
        };
}
```

But the MonoDevelop / VS for Mac misinterprets the curly braces and insists on indenting it as:

```csharp
public MyClass
{
    public static int[] Field =
        new int[]
    {
        42
    };
}
```

While I can correct this myself, it gets very frustrating when editing items in the array. Any time you press enter or add a semicolon to something inside the braces it "corrects" it (which makes sense if it is actually the correct indentation). For example, after the 42 in the first code snippet, add a comma, newline, and another item. It will re-indent both of those lines and you have to manually tab it over again.

I have confirmed that Visual Studio on Windows enforces the indentation in this example correctly (i.e. it follows the first snippet).

> VS bug [#641290](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/641290)

**Comment by xamarin-release-manager** [comment]
Fixed in version 7.8.0.286 (master)

Pull Request #5293 merged by: Lluis Sanchez
Author: mono
Commit: 4edf53d2399bf2e6e79b1d49160a066cd7f1de55 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 7.7.0.1140 (release-7.7)

Pull Request #5293 merged by: Lluis Sanchez
Author: mono
Commit: 4edf53d2399bf2e6e79b1d49160a066cd7f1de55 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 8.6.0.802 (master)

Pull Request #5293 merged by: Lluis Sanchez
Author: mono
Commit: 4edf53d2399bf2e6e79b1d49160a066cd7f1de55 (xamarin/monodevelop)
